pkg/comp-functions/functions/common/billing.go

```diff
@@ -22,6 +23,11 @@ func CreateBillingRecord(ctx context.Context, svc *runtime.ServiceRuntime, comp
 	log := controllerruntime.LoggerFrom(ctx)
 	log.Info("Enabling billing for service", "service", comp.GetName())
 
+	if comp.GetClaimNamespace() == "vshn-test" {
```

Comment:
do we want to parametrize it via component, so it's not that visible to the whole world?

Reply:
That's a better argument :D

However then we have yet another parameter in the already very bloated component.

Reply:
I would still move it there, we will know that it exists instead of forgetting about it. There's no escape from bloating the component. And this is not the solution to un-bloat it. I would really encourage to create a parameter there.

Reply:
I've made it configurable now.

But making a value that could be a constant value configurable just so we don't forget about it also seems wrong. It's not like we remember all the stuff that's in the component as well.

Reply:
I would go to the component to search for smth first instead of the go code.

Reply:
The right place for that would be the docs, but noone ever reads them, so I also stopped documenting...

Reply:
that's also true

Reply:
my idea was to set a global commodore parameter, this way we would keep namespace name for ourselfs